caikit/runtime/grpc_server.py

```diff
@@ -335,18 +335,11 @@ def main():
     # Enable signal handling
     handle_terminations = True
 
-    # Assume we want compiled services for now
-    service_source = (
-        ServicePackageFactory.ServiceSource.GENERATED
-        if caikit_config.runtime.service_generation.enabled
-        else ServicePackageFactory.ServiceSource.COMPILED
-    )
-    log.debug("Running with service source: %s", service_source)
-
+    # We use only generated services for service generation
```

Comment:
I think this can be removed. For folks not familiar with the history of this removal, this would be a confusing comment